### PR TITLE
Fix issue where the stem of multi-part question are being stripped WB…

### PR DESCRIPTION
**Pull request by pooja31chaudhary**

### CODE REVIEW

**Comment by michaelsharman** [comment]
I've done a functional review of this and found the following issue, for `QTI2.1_Sample_Packet/qti_MMP000006.xml`.

The stimulus for part 2 of the item includes the question stem (stimulus) for part one, so you have the following incorrect JSON:

```
{
    "stimulus": "<div><span class=\"class1\"><span class=\"class2\">This task has two parts. Complete <span class=\"class1\"><span class=\"class2\"><span class=\"class3\">both</span></span></span> parts of the task.<br><br></span></span><span class=\"class1\"><span class=\"class2\">Part 1<br><br></span></span><div>\n<span class=\"class1\"><span class=\"class2\">The ordered pairs for three vertices of a square are shown below.<br><br></span></span><div class=\"class19\"><span class=\"class1\"><span class=\"class2\"><math><mrow><mo>(</mo><mn>2</mn><mtext>,</mtext><mo>−</mo><mn>3</mn><mo>)</mo><mo>,</mo><mtext> </mtext><mo>(</mo><mn>2</mn><mo>,</mo><mtext> </mtext><mn>1</mn><mo>)</mo><mo>,</mo><mtext> </mtext><mo>(</mo><mn>6</mn><mtext>,</mtext><mo>−</mo><mn>3</mn><mo>)</mo></mrow></math><br><img src=\"https://assets.learnosity.com/organisations/1/152778.jpg\" alt=\"\" width=\"280\" height=\"283\"></span></span></div>\n<span class=\"class1\"><span class=\"class2\"> <br><br></span></span><span class=\"class1\"><span class=\"class2\">What is the ordered pair for the fourth vertex?</span></span>\n</div>\n<span class=\"class1\"><span class=\"class2\"> <br><br></span></span><span class=\"class1\"><span class=\"class2\">Part 2<br><br></span></span><div><span class=\"class1\"><span class=\"class2\">What is the length of one side of the square?</span></span></div>\n</div>"
}
```

Where you should have:
```
{
    "stimulus": "<span class=\"class1\"><span class=\"class2\"> <br><br></span></span><span class=\"class1\"><span class=\"class2\">Part 2<br><br></span></span><div><span class=\"class1\"><span class=\"class2\">What is the length of one side of the square?</span></span></div>"
}
```

Note the `<div>` elements wrapping both interactions is not needed at all.

File: src/Processors/QtiV2/In/ItemBuilders/RegularItemBuilder.php [hunk 1]
```diff
@@ -74,17 +76,17 @@ public function map(
         if (!$extraContentHtml->load(QtiMarshallerUtil::marshallCollection($itemBody->getComponents()), false)) {
             throw new \Exception('Issues with the content for itemBody, it might not be valid');
         }
-
+
```
[the remark this answers is not in the document]
Comment (poojachaudhary31) [reply src/Processors/QtiV2/In/ItemBuilders/RegularItemBuilder.php]: Done

File: src/Processors/QtiV2/In/ItemBuilders/RegularItemBuilder.php [hunk 2]
```diff
@@ -21,58 +20,45 @@
...
 
     public function map(
-        $itemReference,
-        ItemBody $itemBody,
-        QtiComponentCollection $interactionComponents,
-        QtiComponentCollection $responseDeclarations = null,
-        ResponseProcessingTemplate $responseProcessingTemplate = null,
-        QtiComponentCollection $rubricBlockComponents = null
-    ) {
+    $itemReference, ItemBody $itemBody, QtiComponentCollection $interactionComponents, QtiComponentCollection $responseDeclarations = null, ResponseProcessingTemplate $responseProcessingTemplate = null, QtiComponentCollection $rubricBlockComponents = null
```
[the remark this answers is not in the document]
Comment (pooja31chaudhary) [reply src/Processors/QtiV2/In/ItemBuilders/RegularItemBuilder.php]: Done

File: src/Processors/QtiV2/In/ItemBuilders/RegularItemBuilder.php [hunk 3]
```diff
@@ -21,58 +20,45 @@
...
-
             // Process <responseDeclaration>
             $responseDeclaration = isset($responseDeclarationsMap[$component->getResponseIdentifier()]) ?
                 $responseDeclarationsMap[$component->getResponseIdentifier()] : null;
-
             $outcomeDeclaration = $this->assessmentItem->getOutcomeDeclarations();
             $mapper = $this->getMapperInstance(
-                $component->getQtiClassName(),
-                [$component, $responseDeclaration, $responseProcessingTemplate, $outcomeDeclaration, $this->organisationId]
+                $component->getQtiClassName(), [$component, $responseDeclaration, $responseProcessingTemplate, $outcomeDeclaration]
```
[the remark this answers is not in the document]
Comment (pooja31chaudhary) [reply src/Processors/QtiV2/In/ItemBuilders/RegularItemBuilder.php]: Done

**Pull request merged**